backend/src/test/java/org/eclipse/tractusx/puris/backend/stock/logic/adapter/ItemStockSammMapperTest.java

```diff
@@ -382,16 +379,16 @@ void test_unmarshalling() {
         // Setup from the suppliers point of view
         Material material = new Material();
         material.setProductFlag(true);
-        material.setOwnMaterialNumber(SUPPLIER_MAT_NUMBER);
         material.setMaterialNumberCx(CX_MAT_NUMBER);
 
         MaterialPartnerRelation mpr = new MaterialPartnerRelation();
         mpr.setPartner(customerPartner);
         mpr.setMaterial(material);
         mpr.setPartnerBuysMaterial(true);
-        mpr.setPartnerMaterialNumber(CUSTOMER_MAT_NUMBER);
+        mpr.setPartnerMaterialNumber(SUPPLIER_MAT_NUMBER);
```

Comment:
CUSTOMER_MAT_NUMBER has been correct. ownMaterialNumber above also. Reason: supplier point of view = reportedProductItemStock 
-> material has supplier material number, mpr holds customer's material number